stack_submit.go

```diff
@@ -13,6 +13,8 @@ import (
 
 type stackSubmitCmd struct {
 	submitOptions
+
+	UpdateOnlyDefault bool `config:"stackSubmit.updateOnly" hidden:"" default:"false"`
```

Comment:
I went with `stackSubmit` instead of `submit` since it seems like this configuration would only be useful for the `stack submit` command. But happy to rename or move this whole thing into the `submitOptions` struct.

Reply:
Since this flag should be supported for all multi-branch submit operations, it makes sense to call it submit.updateOnly. (If it was one-command-only, stackSubmit would've made sense.)
I've updated it to do that.